TLM/TLM/Patch/_CitizenAI/_ResidentAI/GetVehicleInfoPatch.cs

```diff
@@ -146,7 +146,11 @@ public class GetVehicleInfoPatch {
                                      () => $"CustomResidentAI.CustomGetVehicleInfo({instanceID}): " +
                                                          $"Force electric! Parked vehicle {parkedVehicleId} is not electric vehicle, wwap with electric one.");
                         // get electric car vehicleInfo
-                        carInfo = Singleton<VehicleManager>.instance.GetRandomVehicleInfo(ref randomizer, service, subService, ItemClass.Level.Level1);
+                        carInfo = Singleton<VehicleManager>.instance.GetRandomVehicleInfo(
+                            ref randomizer,
+                            ItemClass.Service.Residential,
+                            ItemClass.SubService.ResidentialLowEco,
+                            ItemClass.Level.Level1);
```

Comment:
why does it have to be level 1?

Reply:
because all residential vehicles are level1, level2 are car trailers